blog/migrations/0001_initial.py

```diff
@@ -22,6 +20,7 @@ class Migration(migrations.Migration):
                 ('description', models.TextField(max_length=1500)),
                 ('created', models.DateTimeField(auto_now_add=True)),
                 ('image', models.FileField(blank=True, null=True, upload_to='')),
+                ('votes', models.IntegerField(default=0)),
```

Comment:
We should never change a migrations like this. Remove all the changes from this migration file

Reply:
done